Framework/Algorithms/src/AsymmetryCalc.cpp

```diff
@@ -137,36 +137,33 @@ void AsymmetryCalc::exec() {
   Progress prog(this, 0.0, 1.0, blocksize);
   for (size_t j = 0; j < blocksize; ++j) {
     // cal F-aB
```

Comment:
Make a note to consult the following block of code when designing arithmetics for the histogram types.